techniques/html/H99.html

```diff
@@ -153,13 +153,13 @@ <h3>Example 3: Page list in EPUB 2 and DAISY publications</h3>
 &lt;/ncx&gt;</code></pre>
 			</section>
 			<section class="example" id="example-4-go-to-page-mechanism">
-				<h3>Example 4: Go to page mechanism</h3>
-				<p>A mechanism provides the ability to enter a page number as an input and reposition the user to the matching target page location.</p>
+				<h3>事例 4: Go to ページメカニズム</h3>
+				<p>メカニズムにより、入力としてページ番号を記入し、一致するターゲットページの場所に利用者を移動する機能が提供される。</p>
```

Comment:
input と enter がどちらも「入力」と訳される苦しみ……!
「記入」としていただきましたが、「指定」くらいのほうが自然かもしれません。さらにもう少し意訳して、「ページ番号を指定して入力」くらいにしてしまっても良いのではないでしょうか。

事例なので「メカニズムがある」というくらいにしたいです。

```suggestion
				<p>ページ番号を指定して入力すると、一致するターゲットページの場所に利用者を移動させる機能を提供するメカニズムがある。</p>
```